internal/services/notification/webhooks.go

```diff
@@ -39,81 +39,20 @@ const (
 	AgolaEventRun AgolaEventType = "run"
 )
 
-type RunWebhook struct {
```

Comment:
Why are you removing this type? There's a reason to don't use the db type (see the PR where I asked for this).

Reply:
I moved it under internal types package(not db type) because I are using it into migrateV4, so avoid "import cycle not allowed" error.
I can move it under a different package or redefine a RunWebhook into db package if necessary.